src/App.tsx

```diff
@@ -1,26 +1,241 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import {
+  USER_ID,
+  createTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { ErrorNotification } from './components/errorNotification';
+import { Footer } from './components/footer';
+import { Header } from './components/header';
+import { TodoList } from './components/todoList';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
-
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [newTodo, setNewTodo] = useState('');
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState<'all' | 'active' | 'completed'>('all');
```

Comment:
```suggestion
  const [filter, setFilter] = useState<Filter>(Filter.All);
```
You have created enum for it
Actually, you can use union type here instead of enum. but instead of repeating `'all' | 'active' | 'completed'` everywhere you should create such type and reuse

But I'd prefer enum anyway)